torch/_functorch/eager_transforms.py

```diff
@@ -1393,11 +1399,12 @@ def functionalize(func: Callable, *, remove: str = 'mutations') -> Callable:
 
     Example::
 
+        >>> # xdoctest: +SKIP
```

Comment:
We should probably make some spreadsheet for the skips and encourage people to work on them 😝

Reply:
I agree. Fortunately they are very easy to grep for! You can also put a string explanation in them too, which I stopped doing because I've already put a ton of time into this.

In some cases the skip makes sense if the example code isn't meant to be executed, but I do think it's better when you can verify that a piece of code actually executes with some demo information.